cav_srvs/srv/ComputePlanCost.srv

```diff
@@ -0,0 +1,15 @@
+#
+# ComputePlanCost.srv
+# 
+# Provided by the cost plugin node. Returns a final cost of a given maneuver plan.
+#
+# @author Fang Zhou
+# @version 0.1
+#
+
+# Request
+cav_msgs/ManeuverPlan maneuver_plan
+
+---
+# Response
+double plan_cost
```

Comment:
Ah, looks like I'd remembered wrong, double isn't a ROS message type name. `float64` is what you want here.